client/src/controllers/SkipLinkController.test.js

```diff
@@ -0,0 +1,40 @@
+import { Application } from '@hotwired/stimulus';
+
+import { SkipLinkController } from './SkipLinkController';
+
+describe('skip to the main content on clicking the skiplink', () => {
+  document.body.innerHTML = `
+        <a id="skip" class="button" data-controller="w-skip-link" data-action="click->w-skip-link#skipToMain">Skip to main content</a>
+        <main>
+            Main content
+        </main>
+        <button id="other-content">other</button> 
+    `;
```

Comment:
Not a huge problem but we do not need to indent this test HTML so much, also the main/main can be in one line.

This just makes it more readable.